packages/drawer/README.md

```diff
@@ -62,11 +62,11 @@ npm install @material/mwc-drawer
 </mwc-drawer>
 ```
 
-### Dismissable
-<img src="images/dismissable.gif" width="552px">
+### Dismissible
+<img src="images/dismissible.gif" width="552px">
```

Comment:
File needs renaming too (maybe just forgot to add it to the commit?)

Reply:
Woops, thanks for catching that!